docs/core/diagnostics/runtime-garbage-collection-events.md

```diff
@@ -0,0 +1,500 @@
+---
+title: "Garbage Collection Runtime Events"
+description: View detailed information about garbage collection events.
+ms.date: "09/27/2020"
+ms.topic: reference
+helpviewer_keywords:
+  - "GC events"
+  - "garbage collection events [.NET Core]"
+  - "ETW, EventPipe, LTTng garbage collection events (CoreCLR)"
+ms.assetid: f14b6fd7-0966-4d87-bc89-54ef3a44a94a
+---
+# Garbage Collection Events
+
+These events collect information pertaining to garbage collection. They help in diagnostics and debugging, including determining how many times garbage collection was performed, how much memory was freed during garbage collection, etc.
+
+This category consists of the following events:
+
+- [GCStart_V2 Event](#gcstart_v2-event)
+- [GCEnd_V1 Event](#gcend_v1-event)
+- [GCHeapStats_V2 Event](#gcheapstats_v2-event)
+- [GCCreateSegment_V1 Event](#gccreatesegment_v1-event)
+- [GCFreeSegment_V1 Event](#gcfreesegment_v1-event)
+- [GCRestartEEBegin_V1](#gcrestarteebegin_v1-event)
+- [GCRestartEEEnd_V1 Event](#gcrestarteeend_v1-event)
+- [GCSuspendEEBegin_V1 Event](#gcsuspendeebegin_v1-event)
+- [GCSuspendEEEnd_V1 Event](#gcsuspendeeend_v1-event)
+- [GCAllocationTick_V3 Event](#gcallocationtick_v3-event)
+- [GCCreateConcurrentThread_V1 Event](#gccreateconcurrentthread_v1-event)
+- [GCTerminateConcurrentThread_V1 Event](#gcterminateconcurrentthread_v1-event)
+- [GCFinalizersBegin_V1 Event](#gcfinalizersbegin_v1-event)
+- [GCFinalizersEnd_V1 Event](#gcfinalizersend_v1-event)
```

Comment:
This is just a repeat of the H2 headers, these are automatically provided on the right-side of the page.

```suggestion
```